### PR TITLE
Add views for datastream download stats

### PR DESCRIPTION
**Pull request by jonathangreen**
**JIRA Ticket**: 

https://jira.duraspace.org/browse/ISLANDORA-1962

# What does this Pull Request do?

Exposes additional tables from Islandora usage stats to views.

# What's new?

Should just add the additional tables to views.

# How should this be tested?

Install module with pull request and make sure that you can use the new tables in views.

# Interested parties
@willtp87 @bryjbrown

### CODE REVIEW

File: views/islandora_usage_stats.views.inc [hunk 1]
```diff
@@ -67,6 +67,75 @@ function islandora_usage_stats_views_data() {
       ),
     ),
   );
+  $data['islandora_usage_stats_datastreams'] = array(
+    'table' => array(
+      'group' => t('Islandora Usage Stats'),
+      'base' => array(
+        'field' => 'id',
+        'title' => t('Islandora Usage Stats Datastreams'),
+        'help' => t('Datastream information.'),
```
Comment (whikloj) [inline views/islandora_usage_stats.views.inc]: This title makes sense, except its not clear that it is just a table of the datastreams downloaded tied to the PID of the Object. Not sure how to clarify what the purpose is here.

Comment (whikloj) [reply views/islandora_usage_stats.views.inc]: Oh and **help** doesn't seem to appear, at least not in the Garland theme. I just get a select box to choose my data table with the titles.

File: views/islandora_usage_stats.views.inc [hunk 2]
```diff
@@ -168,6 +237,100 @@ function islandora_usage_stats_views_data() {
       ),
     ),
   );
+  $data['islandora_usage_stats_object_ds_access_log'] = array(
+    'table' => array(
+      'group' => t('Islandora Usage Stats'),
+      'base' => array(
+        'field' => 'id',
+        'title' => t('Islandora Usage Stats Object Datastream Access'),
+        'help' => t('The log of Datastreams being visited.'),
```
Comment (whikloj) [inline views/islandora_usage_stats.views.inc]: This title should specify that this is datastream **downloads** and not access. Also the help makes it seem like viewing a datastream, which (perhaps this is just me) I think of as doing when I view the object in Islandora 😄

**Review by whikloj — COMMENTED** [review]
Couple little clarifications, but works as expected...once I knew what to expect :man_facepalming:

**Comment by jonathangreen** [comment]
@whikloj Made some wording changes. Not sure exactly how to describe the datastreams table as well. Let me know what you think.

**Comment by jonathangreen** [comment]
@whikloj bump. do those wording changes look okay?

**Review by whikloj — APPROVED** [review]
Oops sorry about that, approved.

**Pull request merged**